resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -1113,7 +1113,7 @@ function resize() {
             </div>
         @endif
 
-        @if ($numAchievements > 0 && $isOfficial)
+        @if ($numAchievements > 1 && $isOfficial)
```

Comment:
I've noticed this bug as well.

I'm actually wondering if 1 is still too low. Might be worth bumping to `>= 6`?

Reply:
I noticed this when collapsing some of the existing events. Events don't require 6 achievements for a badge, so some of the existing events will be collapsed to having a single achievement after the conversion. A bar chart with a single bar isn't very informative. Other events will be collapsed to one achievement per tier, and a two-tier event would have two bars. Arguably, the tier distribution is handled by #3082, so the limit could be higher.